code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -53,36 +53,39 @@
 		/datum/reagent/oxygen,
 		/datum/reagent/phosphorus,
 		/datum/reagent/potassium,
-		/datum/reagent/uranium/radium,
 		/datum/reagent/silicon,
 		/datum/reagent/silver,
 		/datum/reagent/sodium,
+		/datum/reagent/iron,
+		/datum/reagent/water,
+		/datum/reagent/oil,
```

Comment:
this seems like a very roundabout way to not put welding fuel back on tier 1 even though it does less than oil